scripts/api-compatibility-check/.baseapi

```diff
@@ -1 +1 @@
-v10.1.0
\ No newline at end of file
+81618929724d94fcc64d4be24caed15357e98446
```

Comment:
@macdrevx @OdNairy It seems that because of `MapView.didMoveToSuperview` and `MapView.willMove(toWindow:)` overrides were removed from `MapView` in this PR - it no longer passes the API compatibility check.
Not sure what is the process in this case(it seems the baseline has never been updated since the introduction). I did update the baseline to be the top of this branch. I think one way to go would be to merge this and then update the baseline when next release(or maybe as soon as beta/rc) is out.

Another thing to consider would be to exclude public overrides from API compatibility check. Not sure if `swift-api-digester` supports that(seems not). @OdNairy do you have any insight on this?

Reply:
Should it be considered a breaking change? Seems like not since method dispatch would happen via objc message passing in this case, right? If we wanted, we could leave the overrides in place and have them just call super.

Reply:
This shouldn't be considered a breaking change. I guess, `swift-api-digester` reports false positives for overridden public methods. Let's update the baseline so there are no dangling overrides.